lib/fika/parser/helper.ex

```diff
@@ -39,6 +39,11 @@ defmodule Fika.Parser.Helper do
     {:call, {:!, line}, [exp], :kernel}
   end
 
+  def do_to_ast({[unary_op, exp], line}, :unary_op)
+      when unary_op in ["+", "-"] do
+    {:call, {String.to_atom(unary_op), line}, [exp], :kernel}
```

Comment:
Can it be just `exp` instead of `[exp]` because there's only one exp that will be passed to unary ops?

Reply:
I think it must be a list because otherwise you get an error in the TypeChecker.
This is also true for the boolean not operator.